website/source/layouts/guides.erb

```diff
@@ -50,6 +50,10 @@
         </ul>
       </li>
 
+      <li<%= sidebar_current("guides-tls") %>>
+        <a href="/guides/tls.html">Configuring TLS</a>
```

Comment:
Yeah, I went back and forth a lot on the title and headings that refer to TLS.

Reasons for keeping "tls":
* serf is like 5% of this guide and no one is even thinking about it unless they're already configuring TLS.
* "Securing Nomad" is a much broader topic than this guide covers. There are all kinds of system hardening, config options, etc that go into making Nomad "secure" in a generic sense. I would hate to imply if you follow this guide you've secured Nomad.

Reasons for "securing":
* serf isn't tls
* better seo?
* "Securing Nomad" sounds more appealing

:woman_shrugging: Anybody have a strong opinion?